package.json

```diff
@@ -65,8 +65,11 @@
   },
   "main": "./native/main.comp.js",
   "devDependencies": {
+    "@ethersproject/cli": "^5.0.7",
+    "@rollup/plugin-inject": "^4.0.2",
+    "@rollup/plugin-json": "^4.1.0",
     "@rollup/plugin-commonjs": "^17.0.0",
-    "@rollup/plugin-node-resolve": "^11.0.1",
+    "@rollup/plugin-node-resolve": "^10.0.0",
```

Comment:
Is there a reason we're rolling back, or is this just a merge snafu?
We recently upgraded all packages in master, might be a good idea to run:
```
yarn upgrade-interactive --latest
yarn upgrade
```
just to make sure we're still on the latest & greatest.

Reply:
Yeah, the latest version does not go well with some dependency we are using. trying to remember which one 🤔

Reply:
Might be a good idea to document this somehow, so we don't accidentally upgrade it and break stuff.

Reply:
Or perhaps we just pin it to the specific version, so upgrades don't touch it?

Reply:
If we bump it to `11.x.x.`, we will see a clear error message when running the app. For me, that'd be enough and I assume CI would fail too. Let me know!